### ISSUE TITLE
Can't use .indexedBy

### ISSUE THREAD

**Issue by kurak38** [Raizlabs/DBFlow#1219]
DBFlow Version: 3.1.1
Issue Kind (Bug, Question, Feature): Critical Bug

Description: 
Hi,
I have a crash which produces this error:
`Error Code : 1 (SQLITE_ERROR)
                                                 Caused By : SQL(query) error or missing database.
                                                 	(no query solution (code 1): , while compiling: SELECT * FROM 'SomeTable'INDEXED BY 'someTableIdIndex' WHERE 'someTableId'=3 LIMIT 1)`

Clearly there is 'space' missing before "INDEXED BY" statement. How is it possible that noone notice it?

Also is it possible to turn on indexed queries by default, so I don't need to specify it for every query?
Thx

**Comment by agrosner** [comment]
Probably a bug that has been fixed since then in later 4.0.0-betas.

**Comment by agrosner** [comment]
my recommendation is to try to migrate to 4.0.0-beta6, or Subclassing `IndexedBy` and wrapping it in the query you need. 

```java
new CustomIndexedBy<>(indexProperty, SQLite.select().from(..)).where(..).limit()
```
